### ISSUE TITLE
SD support for REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER?

### ISSUE THREAD

**Issue by gabrielcr78** [MarlinFirmware/Marlin#2800]
Hello, i just started doing some testing with this new version of Marlin 1.1.0-RC2 on a Prusa i3 with RAMPS 1.4

With the previous version of Marlin I had no issue navigating my SD card with the LCD. In fact, if I flash the previous version again it recognizes the SD card immediately. However with this new version, it looks like SD card is not supported?

I uncommented the line for this display just like in my previous Marlin version…

``` cpp
#define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
```

…but not sure if now there is something else that I have to configure for SD support?

Thanks!
Gabe

**Comment by KiteLab** [comment]
``` cpp
//#define ULTRA_LCD //general LCD support, also 16x2
//#define DOGLCD    // Support for SPI LCD 128x64 (Controller ST7565R graphic Display Family)
//#define SDSUPPORT // Enable SD Card Support in Hardware Console
                    // Changed behaviour! If you need SDSUPPORT uncomment it!
```

**Comment by gabrielcr78** [comment]
uhhh thanks a bunch!

**Comment by gabrielcr78** [comment]
thanks!

2016-02-21 19:27 GMT-06:00 Scott Lahteine notifications@github.com:

> Closed #2800 https://github.com/MarlinFirmware/Marlin/issues/2800.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/MarlinFirmware/Marlin/issues/2800#event-558820221.

## 

---

Gabriel Sánchez
